app/pb_kits/playbook/pb_highlight/_highlight.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= content_tag(:span,
+    id: object.id,
+    data: object.data,
+    class: object.classname) do %>
+    <%= !object.children.nil? ? capture(&object.children) : object.text %>
```

Comment:
How about converting this line so it does not use negated logic in order to improve readability?